usr/src/cmd/zdb/zdb.c

```diff
@@ -3529,15 +3529,26 @@ main(int argc, char **argv)
 	char *target;
 	nvlist_t *policy = NULL;
 	uint64_t max_txg = UINT64_MAX;
+	int flags = ZFS_IMPORT_MISSING_LOG;
 	int rewind = ZPOOL_NEVER_REWIND;
+	char *spa_config_path_env;
+	const char *opts = "bcdhilmMI:suCDRSAFLVXx:evp:t:U:P";
```

Comment:
I would suggest that if you're going to move this, you also go ahead and sort it, like https://github.com/openzfs/openzfs/pull/29 proposes.